pkg/kubelet/network/kubenet/bridge.go

```diff
@@ -0,0 +1,159 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubenet
+
+import (
+	"bytes"
+	"fmt"
+	"net"
+	"os"
+	"os/exec"
+	"regexp"
+
+	"github.com/golang/glog"
+	"k8s.io/kubernetes/pkg/util"
+)
+
+var cidrRegexp = regexp.MustCompile(`inet ([0-9a-fA-F.:]*/[0-9]*)`)
+
+func createBridge(name string) error {
+	// recreate the bridge
+	if err := exec.Command("brctl", "addbr", name).Run(); err != nil {
+		glog.Error(err)
+		return err
+	}
+	if err := exec.Command("ip", "link", "set", "dev", name, "mtu", "1460", "up").Run(); err != nil {
```

Comment:
Setting MTU needs to consider the MTU of the "main" interface.  1460 is true for clouds like GCE, but bare metal is probably 1500 and I have no idea what happens in a Jumbo environment.  Maybe a TODO, since this code is migrating towards plugins